client/i18n/locales/english/intro.json

```
@@ -1,6 +1,7 @@
 {
   "responsive-web-design": {
     "title": "Responsive Web Design",
+    "isTranslated": false,
```

```suggestion
    "isTranslated": true,
```

This should be set to `true`??

---

Sorry! Forgot to set that back.